deploy/helm/kuberhealthy/templates/khcheck-deployment.yaml

```diff
@@ -26,6 +31,11 @@ spec:
           memory: 15Mi
         limits:
           cpu: 40m
+{{- if .Values.securityContext.enabled }}
+      securityContext:
+        allowPrivilegeEscalation: {{ .Values.securityContext.allowPrivilegeEscalation }}
+        readOnlyRootFilesystem: {{ .Values.securityContext.readOnlyRootFilesystem }}
+{{- end}}
```

Comment:
any reason why the khcheck-deployment requires the `{{- if .Values.securityContext.enabled }}` statement here versus the other checks?

Reply:
@joshulyne Not sure, but I can add them to all the rest of the other kh checks for consistency as well.